ecosystem/sep-0038.md

```diff
@@ -156,21 +200,20 @@ If the client requests some amount of off-chain currency for providing some amou
 
 In the reverse scenario, the anchor will submit a Stellar transaction to deliver funds to the client as long as the client delivered off-chain funds to the anchor before the expiration. In this case, the anchor will likely increase their margin to cover the cost of submitting the transaction.
 
-This specification could be extended in the future to include functionality where the anchor collects a transaction signature  from the client prior to submitting the transaction, enabling the anchor to have the client pay the network fees and decrease their margin. This feature would increase the technical complexity of the specification and hence will only be added if there is enough support from the ecosystem.
-
 #### Request
 
 Name | Type | Description
 -----|------|------------
 `base` | string | Same as the definition of `base` in `GET /price`.
 `counter` | string | Same as the definition of `counter` in `GET /price`.
 `base_amount` | string | Same as the definition of `base_amount` in `GET /price`.
-`delivery_method` | string | (optional) One of the `name` values specified by the `delievery_methods` array for the relevant currency returned from `GET /info`. If the array is empty, this parameter is not required.
+`expires_after` | [UTC ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) string | (optional) The earliest date and time the client can deliver funds to the anchor. Anchors should return `400 Bad Request` if the requested expiration cannot be provided.
```

Comment:
❗ Because this request is declaring or asking for the quote to expire after a particular time, I think this field should be `expire_after` rather than `expires_after`. `expires_after` sounds like the request for the quote "expire**s** after" that time, when the request is signaling that it wishes the quote that will be delivered to "expire after" the time.
```suggestion
`expire_after` | [UTC ISO 8601](https://www.iso.org/iso-8601-date-and-time-format.html) string | (optional) The earliest date and time the client can deliver funds to the anchor. Anchors should return `400 Bad Request` if the requested expiration cannot be provided.
```